ENH: Reorganize documentation and redirect to nipreps docs (#1367)
nipreps/mriqc: docs/source/usage.rst

Running *MRIQC*
***************
*MRIQC* is a `BIDS-App <http://bids-apps.neuroimaging.io/>`_ [BIDSApps]_,
and therefore it inherently understands the :abbr:`BIDS (brain
imaging data structure)` standard [BIDS]_.
Before moving forward, please make sure to have read and understood
*NiPreps*'s
`introductory documentation <https://www.nipreps.org/apps/framework/>`__).

> Containerized execution with *Docker* and *Singularity*

Reviewer: ```suggestion
Containerized execution with *Docker* and *Singularity*/*Apptainer*
```

Reviewer: I suggest referring to singularity/apptainer with this awkward slash for now. My understanding is that we will eventually call this only "apptainer", but right now things are in a bit of a transition state.

-------------------------------------------------------

> For containerized execution with *Docker* or *Singularity*, please

Reviewer: ```suggestion
For containerized execution with *Docker* or *Singularity*/*Apptainer*, please
```

    In the ``group`` level, the :abbr:`IQMs (image quality metrics)` extracted in
    first place are combined in a table and the group reports are generated.

*MRIQC* can fetch data in *DataLad* datasets
--------------------------------------------

> As of *MRIQC* 22.0.3, the application bundles *DataLad*, enabling automatic

Reviewer: ```suggestion
As of version 22.0.3, *MRIQC* bundles *DataLad*, enabling automatic
```

data fetching in *DataLad* datasets.
Employing this feature in containerized environments may lead to
somewhat obscure errors (see, for example,
`nipreps/mriqc#1307 <https://github.com/nipreps/mriqc/issues/1307>`__).
If you intend to use *DataLad* datasets, please read carefully
*NiPreps*' `help for processing DataLad-managed datasets <https://www.nipreps.org/apps/datalad/>`__.

Alternatively, this feature can be disabled by adding
``--no-datalad-get`` to the command line.
This will separate *DataLad* management from *MRIQC*'s operation,
which can be an effective way of debugging issues and averting

> fault conditions.

Reviewer: Not exactly sure what was intended here, but maybe?

```suggestion
erroneous conditions.
```
